board.py

```diff
@@ -1,3 +1,20 @@
+import enum
+
+EMPTY_SPOT = '.'
+
+class Directions(enum.Enum):
+    DOWN = 1
+    RIGHT = 2
+
+    @classmethod
+    def from_str(cls, direction_str):
+        if direction_str.lower() in {'down', 'd'}:
```

Comment:
hadn't thought of lower/upper good catch

Reply:
for realz